src/index.html

```diff
@@ -15,8 +15,94 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          class="img"
+          src="./images/logo.png"
+          alt="MOYO img"
+        />
+      </a>
+
+      <nav>
+        <ul class="navigation">
+          <li>
+            <a
+              href="#"
+              class="navbar is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
```

Comment:
Using `href="#"` is not recommended as it can cause unexpected behavior when clicked. Consider using a valid URL or `javascript:void(0);` if the link is meant to be a placeholder.